specification/paths/Returns-v1-return-methods.json

```diff
@@ -26,6 +26,16 @@
         "schema": {
           "type": "string"
         }
+      },
+      {
+        "name": "filter[shipping_and_return]",
+        "in": "query",
+        "description": "When set, the endpoint will return only shipping return methods that offer shipping and return service."
+      },
+      {
+        "name": "filter[label_in_the_box]",
+        "in": "query",
+        "description": "When set, the endpoint will return only shipping return methods that offer label in the box service."
```

Comment:
Yes, intentionally did not set a schema. For me it is enough if the `filter[...]` is set in the query, no value needed.